src/main/java/org/graylog2/gelfclient/GelfConfiguration.java

```diff
@@ -147,6 +151,66 @@ public GelfConfiguration queueSize(final int size) {
     }
 
     /**
+     * Check if TLS option for the transport is enabled.
+     *
+     * @return true if TLS is enabled, false if disabled
+     */
+    public boolean isTls() {
+        return tls;
+    }
+
+    /**
+     * Enable/disable TLS for transport.
+     *
+     * @param tls {@code true} to enable TLS, {@code false} to disable it
+     * @return {@code this} instance
+     */
+    public GelfConfiguration tls(final boolean tls) {
+        this.tls = tls;
+        return this;
+    }
+
+    /**
+     * Get the trust certificate chain file for the TLS connection.
+     *
+     * @return the trust certificate chain file
+     */
+    public File getTlsTrustCertChainFile() {
+        return tlsTrustCertChainFile;
+    }
+
+    /**
+     * Set the trust certificate chain file for the TLS connection.
+     *
+     * @param tlsTrustCertChainFile the trust certificate chain file
+     * @return {@code this} instance
+     */
+    public GelfConfiguration tlsTrustCertChainFile(final File tlsTrustCertChainFile) {
+        this.tlsTrustCertChainFile = tlsTrustCertChainFile;
+        return this;
+    }
+
+    /**
+     * Check if TLS certificate verification is enabled.
+     *
+     * @return true if enabled, false otherwise
+     */
+    public boolean isTlsVerifyCert() {
+        return tlsVerifyCert;
+    }
+
+    /**
+     * Enable/disable TLS certificate verification for transport.
+     *
+     * @param tlsVerifyCert {@code true} to enable TLS certificate verification, {@code false} otherwise
+     * @return {@code this} instance
+     */
+    public GelfConfiguration tlsVerifyCert(final boolean tlsVerifyCert) {
```

Comment:
Maybe a method pair like `enableTlsVerification()` and `disableTlsVerification()` would fit better in the builder-like usage of this class.